### PR TITLE
Create news feed

### PR DESCRIPTION
**Pull request by Domianidze**

### CODE REVIEW

File: hooks/useNewsFeedPage.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      1000
+    );
+  };
+
+  const closeModalHandler = () => {
+    router.push('/news-feed');
+  };
+
+  useEffect(() => {
+    window.addEventListener('scroll', scrollHandler);
```
Comment (GigaGiorgadze) [inline hooks/useNewsFeedPage.tsx]: add clean up function to remove event listener

File: types/next-auth.d.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import 'next-auth';
 declare module 'next-auth' {
   interface Session {
     user: {
+      id: string;
```
Comment (GigaGiorgadze) [inline types/next-auth.d.ts:6]: oh just noticed file name in camelCase ik why u wrote in kebab case but library name doesn't matter, or is this must for package? im not sure, if its not change it to camelCase

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]

File: types/next-auth.d.ts (hunk 2 above)
Comment (Domianidze) [reply types/next-auth.d.ts:6]: I checked and it didn't work when I used camelCase so I think it's a must.

Comment (GigaGiorgadze) [reply types/next-auth.d.ts:6]: well that stupid =_= okay

**Review by GigaGiorgadze — APPROVED** [review]

**Pull request merged**